fix: MD007/ul-indent

nschonni opened the pull request:
- Unordered list indentation
- Added code fence languages
- Trim trailing/leading spaces

BillWagner commented on docs/framework/data/wcf/streaming-provider-wcf-data-services.md:
```diff
@@ -1,124 +1,136 @@
+
+To provide the [!INCLUDE[ssAstoria](../../../../includes/ssastoria-md.md)] runtime with access to the <xref:System.Data.Services.Providers.IDataServiceStreamProvider> implementation, the data service that you create must also implement the <xref:System.IServiceProvider> interface. The following example shows how to implement the <xref:System.IServiceProvider.GetService%2A> method to return an ins ...
+
+[!code-csharp[Astoria Photo Streaming Service#PhotoServiceStreamingProvider](../../../../samples/snippets/csharp/VS_Snippets_Misc/astoria%20photo%20streaming%20service/cs/photodata.svc.cs#photoservicestreamingprovider)]
```
I'm a bit confused.  Shouldn't the space characters change to `_` to match dotnet/samples#810?  

And, that begs the question: will other docs need to be updated for those folder and file name changes?

BillWagner replied:
I just saw #11784 

If I understand correctly, we should merge this in the following order:
#11783 
dotnet/samples#810
#11784 

Does that make sense @nschonni ?

nschonni replied:
Thanks, I thought I had undone all of these to handle them separately. I've rebased these ones out so those other PRs should be independent again

nschonni committed:
fix: MD007/ul-indent

- Unordered list indentation
- Added code fence languages
- Trim trailing/leading spaces

nschonni commented:
@BillWagner I've rebased this now that the other PRs landed

nschonni commented:
@BillWagner anything to change here?

BillWagner commented:
@nschonni Thanks for the ping. I was OOF for a long weekend, and this dropped off my radar. Reviewing now.

BillWagner reviewed (approved):
Thanks again for all the work @nschonni 

This is great. I'll :shipit: now.

nschonni commented:
@BillWagner not a problem
